lib/PuppeteerSharp.Tests/InputTests/InputTests.cs

```diff
@@ -40,15 +40,17 @@ public InputTests(ITestOutputHelper output) : base(output)
             Assert.Null(await Page.EvaluateExpressionAsync("result.check"));
             await Page.ClickAsync("input#agree");
             Assert.True(await Page.EvaluateExpressionAsync<bool>("result.check"));
+            var res = await Page.EvaluateExpressionAsync<string[]>("result.events");
             Assert.Equal(new[] {
                "mouseover",
                "mouseenter",
                "mousemove",
                "mousedown",
                "mouseup",
                "click",
+               "input",
                "change"
-            }, await Page.EvaluateExpressionAsync<string[]>("result.events"));
+            }, res);
             await Page.ClickAsync("input#agree");
```

Comment:
for some reason `res` is missing `"input"`, not sure why 😢

Reply:
Try updating chromium https://github.com/kblok/puppeteer-sharp/pull/320/files#diff-0473138966d94a54e5856d4d4befea4fR30

Reply:
You can pull from master now.